lib/riemann/tools.rb

```diff
@@ -65,6 +65,7 @@ def report(event)
       end
       
       event = event.merge(attributes)
+      event[:time] = Integer(event[:time]) if event.has_key? :time
```

Comment:
Well the problem is it *won't* raise: you'll just silently generate weird events. I would be in favor of an assertion that event[:time] is numeric, though.